.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: Mark stale issues
+
+on:
+  schedule:
+  - cron: "0 */12 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
```

Comment:
what is this? runs-on?

Reply:
Not exactly sure, seems to be the default for running any github actions